tensorflow_serving/model_servers/tensorflow_model_server_test.py

```diff
@@ -163,7 +205,41 @@ def _TestBadModelUpconvertedSavedModel(self):
   def _TestBadModelSessionBundle(self):
     """Test Predict against a bad SessionBundle model export."""
     self._TestBadModel(use_saved_model=False)
-
-
+    
+  def testGoodFileConfig(self):
```

Comment:
Can you name it "testGoodModelConfig" (since there are multiple kinds of configs). Thanks.